combat-trainer.lic

```diff
@@ -3501,6 +3514,7 @@ class GameState
   end
 
   def sheath_whirlwind_offhand
+    return if weapon_skill.include?'Twohanded'
```

Comment:
To clarify for myself, this check is here because there wouldn't be an offhand to sheath if you're holding a twohanded wepaon, right?